test/dlc_tests/ec2/tensorflow/training/test_tensorflow_training.py

```diff
@@ -162,3 +166,31 @@ def test_tensorflow_addons_cpu(tensorflow_training, ec2_connection, cpu_only):
     if is_tf1(tensorflow_training):
         pytest.skip("This test is for TF2 only")
     execute_ec2_training_test(ec2_connection, tensorflow_training, TF_ADDONS_CMD)
+
+@pytest.mark.model("dataservice")
+@pytest.mark.parametrize("ec2_instance_type", TF_EC2_CPU_INSTANCE_TYPE, indirect=True)
+def test_tensorflow_dataservice_cpu(tensorflow_training, ec2_connection, cpu_only):
+	if is_tf1(tensorflow_training):
```

Comment:
nit: is the only difference between this function and the function below the fact that one is parametrized as cpu and the other as gpu instance? if so, we could make a private function to house that logic, and call that function in each of the tests, so that we minimize code duplication

Reply:
Added helper function run_data_service_test